src/components/VFilters/VLicenseExplanation.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="license-explanation w-80 p-6">
```

Comment:
```suggestion
  <div class="license-explanation w-full max-w-80 p-6">
```

That's a bit friendlier to the 300-ish pixel screens we've recently improved. We should probably discuss and codify our device size support sometime!